Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -654,15 +654,19 @@ extension ResourceFileElements {
         path: AbsolutePath,
         excluding: [String]
     ) -> Self? {
-        var resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
-        if sources == nil {
-            // SPM automatically includes resources only if custom sources are not specified
-            resourcesPaths += defaultResourcePaths(from: path)
-        }
-        guard !resourcesPaths.isEmpty else { return nil }
 
-        return .init(
-            resources: resourcesPaths.map { absolutePath in
+        func handleProcessRule() -> [ProjectDescription.ResourceFileElement] {
+            let processResources = resources.filter { $0.rule == .process }
+
+            var resourcesPaths = processResources.map { path.appending(RelativePath($0.path)) }
+
+            if sources == nil {
+                // SPM automatically includes resources only if custom sources are not specified
+                resourcesPaths += defaultResourcePaths(from: path)
+            }
```

Comment:
This has nothing to do with `handleProcessRule`, I would handle it separately

Reply:
Moved handling of default resources out of `handleProcessRule` and handle them separately